authors.md

```diff
@@ -2,7 +2,7 @@
 
 * Maximilian Bischoff <mbischoff@inovex.de>
 
-# Contributors
+## Contributors
```

Comment:
Also change the `Initial Author` to be `h1` and add `Authors` as tilte?

Reply:
`Initial Author` is h1 ?

Reply:
But I think I know what you mean.